doc/classes/Control.xml

```diff
@@ -550,9 +550,11 @@
 			</description>
 		</method>
 		<method name="grab_focus">
-			<return type="void" />
+			<return type="bool" />
+			<param index="0" name="p_auto_set_viewport_active_focus_layer" type="bool" />
 			<description>
 				Steal the focus from another control and become the focused control (see [member focus_mode]).
+				Returns [code]true[/code] if it was able to grab focus. Would return [code]false[/code] if the control isn't set to be focused (see [member focus_mode]).
```

Comment:
Doesn't it also return `false` if the controls is on a wrong layer? It should be mentioned too.